tox.ini

```diff
@@ -42,27 +49,29 @@ commands =
     py{35,36,37,38,39,310}:     python -m pytest {env:CI_PYTEST_SPLIT_ARGS:} -n={env:CI_PYTEST_PARALLEL:4} --durations=20 --junitxml=test-results/junit.xml --cov-config=.coveragerc --cov --cov-report= --no-cov-on-fail {posargs:tests/}
     pylaunch:               python -m pytest {env:CI_PYTEST_SPLIT_ARGS:} -n={env:CI_PYTEST_PARALLEL:4} --durations=20 --junitxml=test-results/junit.xml --cov-config=.coveragerc --cov --cov-report= --no-cov-on-fail {posargs:tests/tests_launch/}
 
-[testenv:unit-s_{nb}-py{27,35,36,37,38,39,310}]
+[testenv:unit-s_{nb,kfp}-py{27,35,36,37,38,39,310}]
```

Comment:
do we want 27/35?

Reply:
we arent running them, they are just in the config like the rest of tox.ini.

maybe someone wants to run 27 or 35, i wasnt going to stop them :)